interface/main/tabs/main.php

```diff
@@ -265,5 +435,15 @@ function isEncounterLocked( encounterId ) {
         });
     });
 </script>
+<script>
+    // $('#username').hover(function() {
+        // $('.hideaway').show();
+    // });
+   // $('#username').on ('click', function() {
+        // $('.hideaway').toggle('1000');
+        // $(this).parent().parent().parent().focus();
+    // });
+</script>
```

Comment:
will need to remove this at some point. guessing this is being used for dev debugging.